etc/bash_completion.d/mock

```diff
@@ -4,6 +4,8 @@ _mock_root()
 {
     COMPREPLY+=( $( compgen -W "$( command ls ${1:-/etc/mock} 2>/dev/null | \
         sed -ne 's/\.cfg$//p' )" -X site-defaults -- "$cur" ) )
+    COMPREPLY+=( $( compgen -W "$( command ls -d "$cur"*{.cfg,/} 2>/dev/null | \
+        sed 's:/$::' )" -X site-defaults -- "$cur" ) )
```

Comment:
You're probably right, I've never used compgen before. I'll test it and re-push.